### DISCUSSION TITLE
Requesting test funds

### DISCUSSION THREAD

**Discussion by Ikhsanrekt** [entropyxyz/community#43] [Get Test Funds] [answered]
### Address

5CnkVLs4ayhb9LsoX4E4P7AkmZmuJc2oqQx5pgJhHPgsZUqP

**Comment by Ikhsanrekt** [comment]
Please send it again sir, yesterday my address disappeared

**Comment by johnnymatthews** [comment, marked as answer]
Sent!

> yesterday my address disappeared

Do you mean that you lost your address? [You can export your account from the CLI using this process in future](https://docs.entropy.xyz/guides/manage-accounts#export-an-account).

**Comment by johnnymatthews** [comment]
No response, so I'm gonna close this one.

**Comment by Ikhsanrekt** [comment]
ok
